### ISSUE TITLE
xzomit mothers don't drop organs 100%

### ISSUE THREAD

**Issue by godmode** [EdenServer/community#2292]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x] I have provided my client version in the details. (type /ver into your game window)

### Details
ver 3018_1205

Kill a "mother" xzomit, they should drop a xzomit organ 100% of the time. currently the drop rate is like their children.

**Comment by VirtuosusFFXI** [comment]
Just curious, does this imply that the mother xzomits are currently implemented but simply don't have the correct drop rate? From my observations trying to farm xzomit organs thus far, I have been unable to even locate the mothers in the first place. Am I missing them? From what I recall from retail, there was usually a larger model Xzomit "mother" which a few "baby" Xzomits in tow flying all over the place.

**Comment by VirtuosusFFXI** [comment]
I'm guessing Xzomits such as these are meant to be a "family" and perhaps the mother has simply been disable for the time being?

![Virtuosus_2020 01 03_221446](https://user-images.githubusercontent.com/49970272/71759131-7dcfa400-2e76-11ea-87ae-2fbd908a28b4.png)

That's enough theories from me, just thought I would document it for anyone else curious.

**Comment by Volkaru** [comment]
You are correct that on retail, a larger Xzomit 'mother' would have 1-2 trailing behind her. If you attack a mother, the others link, if you attack a mini, they won't link. It's possible that they worked on first spawn, but after the minis and mother were killed, they just randomly spawned as regular mobs.

**Comment by slassen** [comment]
I should have clarified why this was left the way it was when I wrote the report. Mothers also do not roam with their children following / linking like they should so the drop rate will be left the same as the children until that behavior is added.

**Comment by 59blargedy** [comment]
resolved in November 21, 2022 update
